src/share/poudriere/testport.sh

```diff
@@ -205,10 +205,13 @@ fi
 injail /usr/bin/make -C ${PORTSDIR}/${ORIGIN} maintainer ECHO_CMD=true || \
     err 1 "Port is broken"
 
+__MAKE_CONF=$(mktemp -t poudriere-make.conf)
+setup_makeconf ${__MAKE_CONF} "${JAILNAME}" "${PTNAME}" "${SETNAME}"
```

Comment:
These lines are only needed inside the `if` block below. And then the `__MAKE_CONF` file should be deleted after `make` is ran.

Reply:
Done